package.json

```diff
@@ -54,14 +54,17 @@
     "express-validator": "^6.3.1",
     "joi": "^14.3.1",
     "jsonwebtoken": "^8.5.1",
+    "moment": "^2.24.0",
     "pg": "^7.17.1",
     "sequelize": "^5.21.3",
     "uuid": "^3.4.0"
   },
   "nyc": {
     "include": "server",
     "exclude": [
-      "build", "**/index.js"
+      "build",
+      "**/index.js",
+      "**/db"
```

Comment:
Why did you exclude the db folder in nyc

Reply:
to stop the previously failing coverage caused by seeders. After all, we don't need to test anything in the db folder. Do we?